mozilla/activity-stream
# Bug 1545503 – Improve image quality

Testing: 

- Try some layout variants (recommend: `66-beta-2-complex`, `basic`, `dev-test-all`).
- Ensure image quality looks good
